payments_service/auth/views.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from django.core.exceptions import ObjectDoesNotExist
+
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from slumber.exceptions import HttpClientError
+
+from .. import solitude
+from ..base.views import error_400
+from .forms import SignInForm
+
+log = logging.getLogger(__name__)
+
+
+@api_view(['POST'])
+def sign_in(request):
+    form = SignInForm(request.DATA)
+
+    if not form.is_valid():
+        return error_400(response=form.errors)
+
+    fxa_uuid = u'fxa:{u}'.format(u=form.cleaned_data['access_token']['user'])
```

Comment:
This will create new buyers in Solitude based on the reliable FxA ID. However, this means purchase records will not be linked to marketplace users since those buyers are created from hashed emails: https://github.com/mozilla/webpay/blob/master/webpay/auth/utils.py#L45